xeth/whisper.go

```diff
@@ -36,7 +36,7 @@ func (self *Whisper) Post(payload string, to, from string, topics []string, prio
 			TTL:    time.Duration(ttl) * time.Second,
 			To:     crypto.ToECDSAPub(common.FromHex(to)),
 			From:   key,
-			Topics: whisper.TopicsFromString(topics...),
+			Topics: whisper.NewTopicsFromStrings(topics...),
```

Comment:
I think I saw this style in the stdlibs at a certain point. But I may be mistaking. I think we should follow that. I'll try and look up an example in there.